src/components/Hero/mobile.styles.ts

```diff
@@ -0,0 +1,93 @@
+import { theme } from '@/theme';
+import { getTaglineIconBase, ICON_MIN_SIZE_MOBILE } from './icon.utils';
+import type { HeroStyle, HeroStylesStructure } from './shared.styles';
+
+export const mobileOverrides: Partial<HeroStylesStructure> = {
+  container: {
```

Comment:
Check the comment above inside the desktop style file, and apply the same comments in here as well